### ISSUE TITLE
Focus raises editor

### ISSUE THREAD

**Issue by lesgrieve** [microsoft/vscode#126044]
<b>Bug</b>

vscode 1.57.0 has a breaking change from 1.56.2. On Mac, I use yabai for focus follows mouse. Starting with vscode 1.57.0, the application window is raised to the front every time it gains focus. I checked Settings for a related option but did not find any. I have been able to disable automatic updates and install 1.56.2 successfully.

VS Code version: Code 1.56.2 (Universal) (054a9295330880ed74ceaedda236253b4f39a335, 2021-05-12T17:44:30.902Z)
OS version: Darwin x64 20.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 3|
|Memory (System)|16.00GB (1.84GB free)|
|Process Argv|--crash-reporter-id 8eb56bb7-cf61-4b78-90ab-3204b0eeca02|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (34)</summary>

Extension|Author (truncated)|Version
---|---|---
insert-unicode|bru|0.12.0
vscode-eslint|dba|2.1.22
tabspacerselect|dli|1.0.3
githistory|don|0.6.16
simple-alignment|ear|1.0.1
selected-lines-count|gur|1.4.0
vscode-ts-whitespace-converter|joh|1.2.1
vscode-insertdatestring|jsy|2.3.0
eml|lei|0.4.0
paste-indent|les|0.0.1
git-graph|mhu|1.30.0
vscode-docker|ms-|1.13.0
python|ms-|2021.5.926500501
vscode-pylance|ms-|2021.6.1
jupyter|ms-|2021.6.999230701
remote-containers|ms-|0.183.0
vscode-typescript-tslint-plugin|ms-|1.3.3
java|red|0.79.2
vscode-commons|red|0.0.6
vscode-yaml|red|0.19.2
vscode-fileutils|sle|3.4.5
sonarlint-vscode|Son|2.0.0
rewrap|stk|1.14.0
code-spell-checker|str|1.10.2
sort-lines|Tyr|1.9.0
indent-one-space|use|0.2.6
vscodeintellicode|Vis|1.2.14
insertnums|vol|0.6.4
vscode-java-debug|vsc|0.34.0
vscode-java-dependency|vsc|0.18.4
vscode-java-pack|vsc|0.16.0
vscode-java-test|vsc|0.30.0
vscode-maven|vsc|0.31.0
vscode-ansible|zbr|0.4.5

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt678:30270856
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
pythonvspyt639:30300192
pythontb:30283811
pythonvspyt551cf:30311713
vspre833cf:30267465
pythonptprofiler:30281270
vshan820:30294714
vscoreces:30290705
pythondataviewer:30285071
vscus158:30286553
vscgsv2:30307504
vscorehov:30309549
vscod805:30301674
binariesv517cf:30312826

```

</details>

**Comment by lesgrieve** [comment]
Excellent news, thank you for your prompt response.

**Comment by lesgrieve** [comment]
This issue has been resolved by the 1.57.1 update, than you!
